spa/src/pages/Connected/index.tsx

```diff
@@ -47,18 +43,16 @@ const sectionStyle = css`
 const sectionImgStyle = css`
 	height: 100px;
 `;
-const linkStyle = css`
-	cursor: pointer;
-`;
 
 const buttonStyle = css`
-	padding-left: 0;
-	height: 14px;
-	line-height: 14px;
-	display: inline-flex;
+	margin: 0px 10px;
+`;
+const subtleBtnStyle = css`
```

Comment:
have applied !important to the color prop of the Atlas Toolkit button, because According to the UX provided to us, the button with the appearance of 'subtle' cannot be used as is, as its color looks a bit darker than what is shown in Figma.
and the original Button comp has css color prop with !important, so cant override it with normal css specificity!

Please suggest a solution.